homework-submission/React_HW1/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Welcome to React World</title>
+  <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+  <script src="https://unpkg.com/react@16/umd/react.development.js"></script>
+  <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+
+  <!-- Don't use this in production: -->
+  <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="text/babel">
+    const todoListStatic = [
+        {
+          description: '* Get out of bed',          
+          deadline: 'Wed Sep 13 2017',
+          done: ''
+        },
+        {
+          description: '* Brush teeth',          
+          deadline: 'Thu Sep 14 2017',
+          done: ''
+        },
+        {
+          description: '* Eat breakfast',          
+          deadline: 'Fri Sep 15 2017',
+          done: ''
+        }
+      ];
+      
+         const todoListDynamic = [
+        {
+          id: 1,
+          description: 'Get out of bed',
+          deadline: '2017-09-11',
+          done: true,
+        },
+        {
+          id: 2,
+          description: 'Brush teeth',
+          deadline: '2017-09-10',
+          done: false,
+        },
+        {
+          id: 3,
+          description: 'Eat breakfast',
+          deadline: '2017-09-09',
+          done: false,
+        },
+      ];
+
+// TableTitle Component used by both static and dynamic todo list
+      const TableTitle = () => {
+        return (
+          <thead>
+            <tr>
+              <th>Description</th>
+              <th>Deadline</th>
+              <th>Done</th>
+            </tr>
+          </thead>
+        );
+      }
+
+      class StaticTodoContent extends React.Component {
+        render () {
+          return (
+            <tbody>
+              <tr>
+                <td>{todoListStatic[0].description}</td>
+                <td>{todoListStatic[0].deadline}</td>
+                <td>{todoListStatic[0].done}</td>
```

Comment:
this looks like a static component, I would say it's more to be static content, statics aren't about taking from the parent scope as here, they're about taking from their parents via props.

So this will be a very good example of Components inheritance, 
if you removed this component and tried to use `ContentDynamicTodo` instead in this component's rendering place, it would be as the convention, 

But we could consider the following `Componant` word point's in this case to a class and `content` in my perspective in this case to the `td` element.

And that's why I have called it static content.

Reply:
I try to do i, I hope it is ok.